spec/requests/tutorials/index_spec.rb

```diff
@@ -14,6 +14,8 @@
       end
     end
 
+# rubocop:disable Style/BlockComments
```

Comment:
Ditto, let's delete this block since the new design doesn't have accordions anyway